recipe/meta.yaml

```diff
@@ -1,91 +1,155 @@
 {% set name = "napari" %}
-{% set version = "0.4.14" %}
+{% set version = "0.4.15a" %}
+{% set build = 0 %}
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: napari-meta
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 277dd9492bbdfa2cf5ae60f503abe07bcab851fe69f42bfcbf8d601965f3a92b
+  git_url: https://github.com/napari/napari.git
+  git_rev: conda-bundle-poc
```

Comment:
Reminder: adjust this before merge.

Reply:
Will this go back to PyPI? I am worried about triggering lfs downloads here.

Reply:
Will LFS kick in for github tarballs? I need some files from the repo not present in the pypi tarball.

Reply:
Apparently it's an [opt-in setting](https://github.com/git-lfs/git-lfs/issues/1322#issuecomment-786199434). Can you check whether we have it or not? It we don't use it, I can refer to the github archive tarball. If we do use it, I'll have to assemble things from multiple sources.

Reply:
Ok, I think we are good now. It was only one file in the end! No git sources other than that one file (which is downloaded individually) required.